recipes/wren/build.sh

```diff
@@ -0,0 +1,8 @@
+cmake -DCMAKE_BUILD_TYPE=Release      \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX  \
+      -DCMAKE_PREFIX_PATH=$PREFIX     \
+      -DCMAKE_INSTALL_LIBDIR=lib      \
+      -DSRC_DIR=${SRC_DIR} \
+      ${RECIPE_DIR}
+
+make -j2 install
```

Comment:
`-j${CPU_COUNT}` is better as that will scale to whatever local machine you have etc.